02-express-tutorial/app.js

```diff
@@ -1,55 +1,13 @@
 const express = require("express");
-const { products } = require("./data");
-
+const { getPeople } = require("./controllers/people");
```

Comment:
You should import routes, not controllers. Kindly fix and test your code to ensure it works correctly, as currently only the GET request is functioning.